src/awesome-texmacs.tm

```diff
@@ -38,12 +38,22 @@
 
   <section*|For Users>
 
+  <subsection|Documents written with TeXmacs>
+
```

Comment:
This blank line was unintended. Blank lines are more difficult to control in TeXmacs than in TeX

Reply:
I think is ok.